README.md

```diff
@@ -13,13 +13,13 @@ It also includes the application templates, however they are deprecated in 7.12.
   - [quickstarts](quickstarts): Quickstarts used as example on RHDM images.
   - [scripts](scripts): Holds scripts to help this repository maintenance.
   - [templates](templates): Application templates, used to instantiate a new RHDM environment on OpenShift. (Deprecated)  \
-    - [docs](templates/docs):  Application templates docs generated by the [gen_template_docs.py](https://github.com/jboss-container-images/jboss-kie-modules/blob/master/tools/gen-template-doc/gen_template_docs.py) script.
+    - [docs](templates/docs):  Application templates docs generated by the [gen_template_docs.py](https://github.com/jboss-container-images/jboss-kie-modules/blob/main/tools/gen-template-doc/gen_template_docs.py) script.
 
 Inside each image directory you will find the following files:
 
  - **container.yaml**: used by OSBS builds.
  - **content_sets.yaml**: define the yum repositories needed to install dependencies for the image.
- - **branch-overrides.yaml**: overrides file which uses the latest stable version for external dependencies. The module.yaml and image.yaml always refer to master branch.
+ - **branch-overrides.yaml**: overrides file which uses the latest stable version for external dependencies. The module.yaml and image.yaml always refer to main branch.
```

Comment:
```suggestion
 - **branch-overrides.yaml**: overrides file which uses the latest stable version for external dependencies.
```

Reply:
I see you have resolved it but didn't apply the suggestion, can you please apply it?